source/linux/Demo_User_Guides/TI_LVGL_Demo_User_Guide.rst

```diff
@@ -290,11 +333,22 @@ This widget contains a slide-show on Application Processor Security for AM6X dev
 Building the TI LVGL Demo from Sources
 **************************************
 
-The TI LVGL Demo is enabled in :file:`tisdk-default-image` yocto filesystem for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that
-the binary itself does not have asset images and slides built in it. :file:`tisdk-default-image` contains the required assets within
-:file:`/usr/share/ti-lvgl-demo/*`. Place any additional assets here while making any modifications. Yocto recipe for
-building this demo can be found at
-`github: ti-lvgl-demo.bb <https://github.com/TexasInstruments/meta-tisdk/blob/scarthgap/recipes-demos/ti-lvgl-demo/ti-lvgl-demo.bb>`__
+.. ifconfig:: CONFIG_sdk in ('SITARA')
+
+    The TI LVGL Demo is enabled in :file:`tisdk-default-image` yocto filesystem for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that
+    the binary itself does not have asset images and slides built in it. :file:`tisdk-default-image` contains the required assets within
+    :file:`/usr/share/ti-lvgl-demo/`. Place any additional assets here while making any modifications. Yocto recipe for
+    building this demo can be found at
+    `github: ti-lvgl-demo.bb <https://github.com/TexasInstruments/meta-tisdk/blob/scarthgap/recipes-demos/ti-lvgl-demo/ti-lvgl-demo.bb>`__
+
+.. ifconfig:: CONFIG_sdk in ('DebianSDK')
+
+    The TI LVGL Demo is packaged in :file:`tisdk-debian-trixie` wic image for |__PART_FAMILY_DEVICE_NAMES__| by default. Note, that
+    the binary itself does not have asset images and slides built in it. :file:`tisdk-debian-trixie` contains the required assets within
+    :file:`/usr/share/ti-lvgl-demo/`. Place any additional assets here while making any modifications. Debian package
+    files for building this demo can be found at
+    `github: ti-lvgl-demo debian package <https://github.com/TexasInstruments/debian-repos/blob/master/ti-lvgl-demo/suite/trixie/debian/>`__
+
```

Comment:
Split the common parts of this paragraph into something that can be ifconfiged directly. Too much duplication here.